Sources/SmileID/Classes/Networking/Models/Document.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+public struct Document {
+    public var countryCode: String
+    public var documentType: String
```

Comment:
documentType will be optional for global doc v. since this code is just being introduced and if it would be simple to make this optional from the get-go, we should do that. But it's not a blocker for getting this in